client/client-lib/src/lib.rs

```diff
@@ -5,6 +5,7 @@ pub mod transaction;
 pub mod utils;
 pub mod wallet;
 
+pub mod modules;
```

Comment:
I've put everything into separate module just to get it out of the way, while being able to use existing types if I want to.